CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+#### Version 10.0.0
+* [BREAKING] Added support for using the Thunderhead SDK with Xcode 14.0. The Thunderhead SDK now requires Xcode 14+ . For further details on this [see our readme](https://github.com/thunderheadone/one-sdk-ios#prerequisites).
+
```

Comment:
Need to call out the minimum SDK version 11 as well

Reply:
Updated